packages/dapp/src/views/editBasicProfile/index.tsx

```diff
@@ -0,0 +1,356 @@
+              original: {
+                src: `ipfs://${cids[key]}`,
+                mimeType: "image/*",
+                // TODO: change hardcoded width & height
+                width: 200,
+                height: 200,
+              },
+            };
+          } else {
+            delete values[key];
+          }
+        });
+      }
+
+      if (values["residenceCountry"] === "") {
+        delete values["residenceCountry"];
+      }
+      if (values["birthDate"] === "") {
+        delete values["birthDate"];
+      }
+      if (!imageFile) {
+        delete values["image"];
+      }
+      if (!backgroundFile) {
+        delete values["background"];
+      }
+
+      await mySelf.client.dataStore.merge("basicProfile", values);
+      nextStep();
+    } catch (error) {
+      console.log("Error while saving BasicProfile: ", error);
+      alert("Error while saving data");
+    }
+  };
+
+  return (
+    <Box as="main" w="full">
+      <Stack as="form" onSubmit={handleSubmit(onSubmit)}>
+        <FormControl isInvalid={errors.name}>
+          <FormLabel htmlFor="name">Name</FormLabel>
+          <Input
+            placeholder="anon"
+            borderColor="purple.500"
+            {...register("name", {
+              maxLength: {
+                value: 150,
+                message: "Maximum length should be 150",
+              },
+            })}
+          />
+          <FormErrorMessage>
+            {errors.name && errors.name.message}
+          </FormErrorMessage>
+        </FormControl>
+
+        <FormControl isInvalid={errors.image}>
+          <FormLabel htmlFor="image">Profile Image</FormLabel>
+          <Image ref={image} src={imageURL} />
+          <Input
+            borderColor="purple.500"
+            type="file"
+            defaultValue=""
+            // @ts-expect-error
+            ref={register}
+            placeholder="image"
+            {...register("image")}
+            name="image"
+            onChange={onFileChange}
+          />
+          <FormErrorMessage>
+            {errors.image && errors.image.message}
+          </FormErrorMessage>
+        </FormControl>
+
+        <FormControl isInvalid={errors.background}>
+          <FormLabel htmlFor="background">Header Background</FormLabel>
+          <Image ref={background} src={backgroundURL} />
+          <Input
+            type="file"
+            {...register("background")}
+            borderColor="purple.500"
+            defaultValue=""
+            onChange={onFileChange}
+            placeholder="background"
+          />
+          <FormErrorMessage>
+            {errors.background && errors.background.message}
+          </FormErrorMessage>
+        </FormControl>
+
+        <FormControl>
+          <FormLabel>Description</FormLabel>
+          <Textarea
+            borderColor="purple.500"
+            type={"text"}
+            placeholder="Web3 and Blockchain enthusiast"
+            // value={description}
```

Comment:
Why is it commented?

Reply:
yeah, I don't see any point in this. I'll remove it